Formula/dotnet.rb

```diff
@@ -0,0 +1,77 @@
+class Dotnet < Formula
+  desc ".NET Core"
+  homepage "https://dotnet.microsoft.com/"
+  url "https://github.com/dotnet/source-build.git",
+      tag:      "v3.1.108-SDK",
+      revision: "05a0c8f6b13fcbec4b441c977b639b75e8a74f67"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on xcode: :build
+  depends_on "curl"
+  depends_on "icu4c"
+  depends_on "openssl"
+
+  # Patch of https://github.com/dotnet/source-build/pull/1789 which will be
+  # released with tag 3.1.110 in November 2020.
+  resource "0005-Fix-bad-configure-tests.patch" do
+    url "https://github.com/dotnet/source-build/raw/release/3.1/patches/coreclr/0005-Fix-bad-configure-tests.patch"
```

Comment:
You should use https://raw.githubusercontent.com/dotnet/source-build/17c6409189ed29f0fac2e8f4b1c30d882e6756b5/patches/coreclr/0005-Fix-bad-configure-tests.patch instead to make sure this link doesn't become a 404 once the patch is removed in `release/3.1`.